be/src/vec/olap/vgeneric_iterators.cpp

```diff
@@ -166,6 +166,15 @@ class VMergeIteratorContext {
         if (cmp_res != 0) {
             return cmp_res > 0;
         }
+        
+        if (_sequence_id_idx != -1) {
+            int col_cmp_res = this->_block.compare_column_at(_index_in_block, rhs._index_in_block, _sequence_id_idx, rhs._block, -1);
+            auto left_col = this->_block.get_by_position(_sequence_id_idx).column;
```

Comment:
the `left_col` and `right_col` look like unless， maybe not need to add method of `_block.compare_column_at`. direct use left_col method `compare_at` function look more pretty？

Reply:
👌🏻